main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -3126,8 +3126,11 @@ object Weeder2 {
                   val error = MismatchedTypeParameters(tparamsTree.loc)
                   sctx.errors.add(error)
                   tparams
+                // No type parameters. Issue an error and return an empty list.
```

Comment:
We usually have these comments under the case no?

Reply:
Not in my experience